### PR TITLE
SASS-6284: Create UK Dividends, Shares and Loans Page

### PR DESCRIPTION
**Pull request by ansahdavid**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the it tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?
- [ ]  Have you done the group review in the team?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the it tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- [ ]  Have you checked for individual and agent content matching with the prototype? (where applicable)
- [ ]  Have you checked for welsh transaltion for individual and agent? (where applicable)
- [ ]  Have you checked for errors messages for individual and agent on both english and welsh? (where applicable)
- [ ]  Run wave (chrome extension) for accessibility (where applicable)

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?
- [ ]  Demo to Business Analyst

### CODE REVIEW

File: app/views/propertypensionsinvestments/UkDividendsSharesLoansAgentView.scala.html [hunk 1]
```diff
@@ -34,14 +80,44 @@
...
+                content = HtmlContent(insetContent4)
+            )
+        )
+
         @govukCheckboxes(
             CheckboxesViewModel(
                 form     = form,
-                idPrefix = "UkDividendsSharesLoans",
+                idPrefix  = "UkDividendsSharesLoans",
                 name     = "value[]",
```
Comment (ghousedev) [inline app/views/propertypensionsinvestments/UkDividendsSharesLoansAgentView.scala.html:117]: Could you align please

File: app/views/propertypensionsinvestments/UkDividendsSharesLoansView.scala.html [hunk 2]
```diff
@@ -34,14 +75,44 @@
...
+                content = HtmlContent(insetContent4)
+            )
+        )
+
         @govukCheckboxes(
             CheckboxesViewModel(
                 form     = form,
-                idPrefix = "UkDividendsSharesLoans",
+                idPrefix  = "UkDividendsSharesLoans",
                 name     = "value[]",
```
Comment (ghousedev) [inline app/views/propertypensionsinvestments/UkDividendsSharesLoansView.scala.html:112]: Could you align please

File: app/views/propertypensionsinvestments/UkDividendsSharesLoansAgentView.scala.html (hunk 1 above)
Comment (ansahdavid) [reply app/views/propertypensionsinvestments/UkDividendsSharesLoansAgentView.scala.html:117]: Thanks for pointing this out @ghousedev, I've corrected this in my latest commits

File: app/views/propertypensionsinvestments/UkDividendsSharesLoansView.scala.html (hunk 2 above)
Comment (ansahdavid) [reply app/views/propertypensionsinvestments/UkDividendsSharesLoansView.scala.html:112]: Thanks for pointing this out @ghousedev, I've corrected this in my latest commits

**Review by basotia — APPROVED** [review]

**Review by ghousedev — APPROVED** [review]

**Pull request merged**